fr/administrator-guide/configuring-the-catalog/inspire-configuration.rst

```diff
@@ -1,4 +1,36 @@
 .. _inspire-configuration:
 
 Configurer pour la directive INSPIRE
-####################################
\ No newline at end of file
+####################################
+
+La configuration d'un service CSW conforme INSPIRE se réalise au niveau de la 
+section "Administration > Paramètres > CSW" comme vu précédemment (voir 
+`Configurer le CSW <./csw-configuration.html>`_). Sur base du formaulaire de 
+paramétrisation du CSW du catalogue, il est impératif que le gestionnaire 
+remplisse les champs suivants:
+
+- Contact - Sélectionner que la personne de contact souhaiter. Vérifier au 
+  préalable que l'ensemble des coordonnées de ce point de contact ont été 
+  complétées afin de se conformer à la directive INSPIRE pour le CSW.
+- Information sur le service - Compléter l'ensemble des champs (titre, résumé, 
+  honoraires, contraintes d'accès) et ce pour l'ensemble des langues référencées
+  au niveau du catalogue pour INSPIRE.
+
+
+.. figure:: img/csw-inspire.png
```

Comment:
Files are named with ".PNG" and figure use ".png". It sounds better to use everywhere lowercase file extension.